spec/Domains.tex

```diff
@@ -958,6 +958,35 @@ \subsection{Methods on All Domain Types}
 \end{chapelprintoutput}
 \end{chapelexample}
 
+\index{domains!dist@\chpl{dist}}
+\index{predefined functions!dist@\chpl{dist}}
+\begin{protohead}
+proc $Domain$.dist() : dmap
```

Comment:
Should it be listed without parentheses here?

Reply:
Yup! I'll fix momentarily.